### ISSUE TITLE
We need an auto resume when Uploading Taxonomic Data and Occurrence data files

### ISSUE THREAD

**Issue by helendallas** [kartoza/django-bims#4132]
When uploading large master lists (taxonomic data) or occurrence data files, the process sometimes stops.

Then we need to cancel and try again.

Sometimes this is due to an error but we never know why it has stopped.

Is there a way to check? or build into the system a feedback mechanism that the uploader receives to give an indication  of the issue, and if it is an connection issue, then it auto resumes uploading.

**Comment by dimasciput** [comment]
Yes, I agree with that, @helendallas. Currently, we have some infrastructure issues that prevent us from holding a long process in the background. So maybe an auto-resume feature is a better solution for now while we figure out the issue.
